src/widget/form/profileform.cpp

```diff
@@ -520,7 +520,10 @@ void ProfileForm::onRegisterButtonClicked()
             break;
         default:
             QString errorMessage = Toxme::getErrorMessage(code);
-            GUI::showWarning(tr("Toxme error"),  errorMessage);
+            qWarning() << errorMessage;
+            QString translated = Toxme::translateErrorMessage(code);
+            if (translated != QString())
```

Comment:
We should still show the default error message here, because else the user might be confused why it didn't work, but showed no message.